ci/test_files_touched.py

```diff
@@ -98,17 +98,29 @@
         'tern report -f json -i photon:3.0',
         'tern report -f spdxtagvalue -i photon:3.0',
         'tern report -f spdxjson -i photon:3.0',
+        'tern report -f spdxyaml -i photon:3.0',
+        'tern report -f spdxxml -i photon:3.0',
+        'tern report -f spdxrdf -i photon:3.0',
         'tern report -d samples/alpine_python/Dockerfile',
         'tern report -f html -i photon:3.0',
         'tern report -f cyclonedxjson -i photon:3.0'],
     # tern/formats/spdx
     re.compile('tern/formats/spdx'): [
-        'tern report -f spdxtagvalue -i photon:3.0 -o spdx.spdx && ' \
-        'java -jar tools-java/target/tools-java-*-jar-with-dependencies.jar '\
+        'tern report -f spdxtagvalue -i photon:3.0 -o spdx.spdx && '
+        'java -jar tools-java/target/tools-java-*-jar-with-dependencies.jar '
         'Verify spdx.spdx',
-        'tern report -f spdxjson -i photon:3.0 -o spdx.json && ' \
-        'java -jar tools-java/target/tools-java-*-jar-with-dependencies.jar '\
-        'Verify spdx.json'],
+        'tern report -f spdxjson -i photon:3.0 -o spdx.json && '
```

Comment:
@armintaenzertng Do these lines need the continuation like the lines that were removed?